tests/i18n/test_extraction.py

```diff
@@ -868,6 +868,32 @@ def test_add_location_gettext_version_check(self):
             )
 
 
+class ObsoleteTranslationTests(ExtractorTests):
+    work_subdir = "obsolete_translations"
+
+    PO_FILE = "locale/%s/LC_MESSAGES/django.po"
+
+    def test_no_obsolete(self):
+        management.call_command(
+            "makemessages", locale=[LOCALE], verbosity=0, no_obsolete=True
+        )
+        self.assertTrue(os.path.exists(self.PO_FILE % LOCALE))
```

Comment:
Use `assertIs(..., True)` instead of `assertTrue()`. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).
```suggestion
        self.assertIs(os.path.exists(self.PO_FILE), True)
```